### ISSUE TITLE
[Data Issue]: Estimate for British Columbia (CA-BC) assumes generation thermal but it's mostly hydro

### ISSUE THREAD

**Issue by seamus-oconnor** [electricitymaps/electricitymaps-contrib#6129]
### When did this happen?

Recently but not sure of the exact date. I checked a couple of days ago and British Columbia has always been a "grey" province with no information other than transmission to Alberta and Washington state.

### What zones are affected?

CA-BC

### What is the problem?

The map https://app.electricitymaps.com/zone/CA-BC shows British Columbia as having a carbon intensity now of:
> 700 gCO₂eq/kWh (Source: assumes thermal (coal, gas, oil or biomass))

British Columbia produces most of it's electricity from hydro. It would be better to use [this data from the federal government of Canada](https://www.cer-rec.gc.ca/en/data-analysis/energy-markets/provincial-territorial-energy-profiles/provincial-territorial-energy-profiles-british-columbia.html) to estimate the unknown CO₂eq emissions instead of assuming it is thermal.

**Comment by jarek** [comment]
This is somehow related to estimates. We don't have realtime data for BC and up until recently it was not shown on the map. We've had yearly generation stats for BC linked several times in the issues tracker, so I assume it was either a mistake to start estimating it, or an oversight.

**Comment by VIKTORVAV99** [comment]
> This is somehow related to estimates. We don't have realtime data for BC and up until recently it was not shown on the map. We've had yearly generation stats for BC linked several times in the issues tracker, so I assume it was either a mistake to start estimating it, or an oversight.

It was not, we created a new estimation model that only looks demand data I believe (could be generation as well, I'll have to double check) but that meant that we can estimate a lot more zones. And while this is not super accurate for those zones in particular it allows them to be flowtraced which improves the accuracy of zones they have exchanges with.

But since this was a general estimation method we implemented it runs on all zones with matching data so it was enabled for all zones at once. Even if they didn't have carbon intensity overrides, this was an oversight on our part. But once added it will be retroactively corrected (eventually).
